package.json

```diff
@@ -35,7 +35,7 @@
         "stats:venia": "yarn venia run build:analyze && yarn venia run stats",
         "storybook:venia": "yarn venia run storybook",
         "test": "node --experimental-worker node_modules/.bin/jest",
-        "test:ci": "node --experimental-worker node_modules/.bin/jest --json --outputFile=test-results.json",
+        "test:ci": "node --experimental-worker node_modules/.bin/jest --max-workers=3 --json --outputFile=test-results.json",
```

Comment:
Setting `--max-workers=3` for `test:ci`. This will limit threads spaw to 3. 3 because these tests are run on machine with 4 CPU cores, 1 for the main thread and 3 for testing workers. This is configuration is recommended by [Jest docs](https://jestjs.io/docs/en/cli#maxworkers-num-string).

After making the change, I have noticed the number of threads spawned was reduced by half (previously 7 on my local machine because it was quad-core dual threaded machine giving it 8 virtual cores, 1 for main thread and 7 workers). Tests take longer but less footprint at the same time. Hopefully, this will fix our AWS resource allocation issue.

Reply:
I don't have access to change the AWS machine settings. Can someone change the settings to use `3 GB memory, 2 vCPUs`? As of now, it is `7 GB memory, 4 vCPUs`.

https://console.aws.amazon.com/codesuite/codebuild/projects/pwa-pr-test/edit/environment?region=us-east-1

![image](https://user-images.githubusercontent.com/35203638/69097344-fb9d3480-0a1b-11ea-94bd-3b89d77e1331.png)